src/main/java/org/example/userproject1/entity/Phone.java

```diff
@@ -0,0 +1,22 @@
+package org.example.userproject1.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Data
+@Table(name = "phone_namber")
+public class Phone {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String phone;
```

Comment:
Consider using more conventional field name here instead of phone (eg phoneNumber)